modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -56,12 +82,25 @@ nameserver=4.4.4.41
 ----
 
 a|Specify multiple network interfaces by specifying multiple `ip=` entries.
+
+If the additional network gateway is different from the primary network gateway, the default gateway must be the primary network gateway.
 a|
 ----
 ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 ip=10.10.10.3::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
 ----
 
+Configure the default gateway: 
+
+----
+ip=::10.10.10.254::::
+----
+Configure the route for the additional network:
+
+----
+rd.route=20.20.20.0/24:20.20.20.254:enp2s0
+---- 
+
```

Comment:
Can we break this section into it's own table row? I don't want to mix up the example of specifying multiple `ip=` entries with providing an additional routing entry.